src/masternode-payments.cpp

```diff
@@ -162,7 +162,7 @@ bool IsBlockValueValid(const CBlock& block, int nBlockHeight, CAmount blockRewar
 
 bool IsBlockPayeeValid(const CTransaction& txNew, int nBlockHeight, CAmount blockReward)
 {
-    if(!masternodeSync.IsSynced() || fLiteMode) {
+    if(fLiteMode) {
```

Comment:
Won't work for superblocks I guess.

Reply:
Hmm, not sure I understand. For SBs, validation would just be skipped and thus succeed...am I missing something?

Reply:
Hmm, the way it is in this PR should work if I'm not missing anything. `IsBlockPayeeValid` only does payee checks and no block value checks. This means, if no SB is found (no matter if in IBD or not), `IsBlockPayeeValid` won't do anything in regard to SBs but still do MN payee checks. If a SB is found, it will verify payees no matter if IBD is done or not (which should be fine).

This is a different thing for `IsBlockValueValid`, which must skip full validation of SBs when not fully synced.

Reply:
Ah, true, ok :)